includes/class-sanitize.php

```diff
@@ -119,4 +119,15 @@ public static function constant_value( $value ) {
 
 		return $value;
 	}
+
+	/**
+	 * Sanitize the max image attachments.
+	 *
+	 * @param string $value The value to sanitize.
+	 *
+	 * @return int The sanitized value.
+	 */
+	public static function max_image_attachments( $value ) {
```

Comment:
this is very specific to the `max_image_attachment`, do we really want to have that in the generic Sanatize class?

Reply:
I was on the fence about it. Wasn't sure we wanted to capture all sanitization callbacks here. I can change it

Reply:
Fixed in [967ba2f](https://github.com/Automattic/wordpress-activitypub/pull/1631/commits/967ba2f3dff7fe9cb477c28cd76ded397e4915eb).